packages/core/package.json

```diff
@@ -132,7 +146,8 @@
   "devDependencies": {
     "ajv": "^8.17.1",
     "bunchee": "5.3.2",
-    "natural": "^8.0.1"
+    "natural": "^8.0.1",
+    "python-format-js": "^1.4.3"
```

Comment:
@himself65 I think that the `python-format-js` should be in `dependencies` and not in `devDependencies`. 

I'm saying this because it's being used at  `packages/core/src/prompts/base.ts`.

Can you please confirm?